_posts/en/newsletters/2023-04-05-newsletter.md

```diff
@@ -129,17 +133,19 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
   regular forwarding information and are provided to the spender or
   sender who uses them to send a payment to the first of the hops.  That
   hop begins the process of decrypting the next hop, forwarding the
-  payment to it, having that hop decrypted the subsequent hop, etc,
+  payment to it, having that hop decrypt the subsequent hop, etc,
   until the receiver accepts the payment without their node being
   disclosed to the spender or sender.
 
 {% include references.md %}
 {% include linkers/issues.md v=2 issues="5967,2566,2062,1031,1032,1040,2125,4826,4782,4799,765" %}
-[lnd v0.16.0-beta.rc5]: https://github.com/lightningnetwork/lnd/releases/tag/v0.16.0-beta.rc5
+[lnd v0.16.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.16.0-beta
 [bdk 1.0.0-alpha.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-alpha.0
 [segura watchtowers post]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-March/003892.html
 [segura watchtowers gist]: https://gist.github.com/sr-gi/f91f007fc8d871ea96ead9b27feec3d5
```

Comment:
nit: given all other references have been updated from segura to delgado, these may also need updating.